src/community/advanced-data-card.tsx

```diff
@@ -0,0 +1,462 @@
+                                    hyphens="auto"
+                                >
+                                    {pretitle}
+                                </Text2>
+                            )}
+                            <Text4
+                                color={vars.colors.textPrimary}
+                                truncate={titleLinesMax}
+                                weight="regular"
+                                as={titleAs}
+                                hyphens="auto"
+                            >
+                                {title}
+                            </Text4>
+                            <Text2
+                                color={vars.colors.textPrimary}
+                                truncate={subtitleLinesMax}
+                                as="p"
+                                regular
+                                hyphens="auto"
+                            >
+                                {subtitle}
+                            </Text2>
+                        </Stack>
+                    </Stack>
+                </header>
+
+                {description && (
+                    <Text2
+                        truncate={descriptionLinesMax}
+                        as="p"
+                        regular
+                        color={vars.colors.textSecondary}
+                        hyphens="auto"
+                    >
+                        {description}
+                    </Text2>
+                )}
+            </Stack>
+        </>
+    );
+};
+
+type CardFooterProps = {
+    button?: RendersNullableElement<typeof ButtonPrimary>;
+    footerImage?: RendersNullableElement<typeof Image>;
+    footerText?: string;
+    footerTextLinesMax?: number;
+    buttonLink?: RendersNullableElement<typeof ButtonLink>;
+};
+
+const CardFooter: React.FC<CardFooterProps> = ({
+    button,
+    footerImage,
+    footerText,
+    footerTextLinesMax,
+    buttonLink,
+}) => {
+    const hasButton = !!button;
+    const hasFooterImage = !!footerImage;
+    const hasFooterText = !!footerText;
+    const hasButtonLink = !!buttonLink;
+    const hasAllItens = hasButton && (hasFooterImage || hasFooterText) && hasButtonLink;
+
+    const maxWidth = hasButtonLink && !hasAllItens ? '178px' : '';
+
+    return (
+        <>
+            <div className={styles.adjustmentDivider}>
+                <Divider />
+            </div>
+
+            <div
+                className={classNames(
+                    styles.actions,
+                    styles.actionsVariants[hasAllItens ? 'mobile' : 'default']
+                )}
+            >
+                {hasButton && (
+                    <div
+                        tabIndex={-1}
+                        className={classNames(
+                            sprinkles({
+                                display: 'flex',
+                            }),
+                            styles.marginTopButton
+                        )}
```

Comment:
instead of merging `sprinkles` + `styles.marginTopButton` here, I'd move the sprinkes styles to the css.ts file and rename `marginTopButton` to just `button`

Reply:
the reason for separating marginTop and marginTopButton is to simplify the logic of the size of the top when it has or not all the elements in the footer, but to separate as much as possible the use of sprinkles with the styles, I made the marginTopButton have the elements repeated in the marginTop and the 'display:flex' which is used wherever it is used.

![image](https://github.com/Telefonica/mistica-web/assets/123959276/cd94ea43-6c57-4824-aafd-a8348424cd03)